source/CMakeLists.txt

```diff
@@ -115,6 +115,7 @@ set(SPIRV_SOURCES
 
   ${CMAKE_CURRENT_SOURCE_DIR}/util/bitutils.h
   ${CMAKE_CURRENT_SOURCE_DIR}/util/hex_float.h
+  ${CMAKE_CURRENT_SOURCE_DIR}/util/parse_number.h
```

Comment:
For newly added files, just remember to have a counterpart PR ready for shaderc.